Automattic/wp-calypso
# Update Form Select arrow icon

Fixes #88203

## Proposed Changes

* Update dropdown icon used in form select to be consistent with the `SelectDropdown` component.
* Removed hover and focus state arrows. The icon color should remain consistent throughout these states.

| Before | After |
|--------|--------|
| <img width="225" alt="Screenshot 2024-03-06 at 10 14 15" src="https://github.com/Automattic/wp-calypso/assets/417538/cf518638-bd63-4860-aeee-a938abc3daff"> | <img width="228" alt="Screenshot 2024-03-06 at 10 13 26" src="https://github.com/Automattic/wp-calypso/assets/417538/2ab38b23-e193-4321-87c6-5e483a18478d"> |

## Testing Instructions



* Run Calypso using `yarn start`
* Navigate to: http://calypso.localhost:3000/devdocs/design/form-fields
* Observe that the icon in the Form Select now matches the one used in the `SelectDropdown` component

## Pre-merge Checklist
